src/pyodide-py/pyodide/_base.py

```diff
@@ -81,12 +81,14 @@ class CodeRunner:
 
     def __init__(
         self,
-        ns: Optional[Dict[str, Any]] = None,
+        globals: Optional[Dict[str, Any]] = None,
+        locals: Optional[Dict[str, Any]] = None,
```

Comment:
You are shadowing the `global` / `locals` objects here, which is not great. Maybe `ns_globals` and `ns_locals`?
You also need to update the docstring above and in `eval_*`

Reply:
Yeah, it's not great. I've seen this done in enough other peoples' code that I've gotten used to it. `ns_globals` seems fine to me. @casatir?

Reply:
@rth Actually, look at the built in `eval` and `exec` functions:
https://docs.python.org/3/library/functions.html#eval
https://docs.python.org/3/library/functions.html#exec
They also call these arguments `globals` and `locals`. I think it's a good idea to name them `globals` and `locals` here too.